gstreamer/src/tags.rs

```diff
@@ -17,6 +17,22 @@ use glib::translate::{from_glib, from_glib_full, ToGlib, ToGlibPtr, ToGlibPtrMut
 use glib::value::{FromValueOptional, SetValue, ToSendValue, TypedValue, Value};
 use glib::StaticType;
 
+#[cfg(feature = "serde-support")]
+use std::ffi::CString;
+
+#[cfg(feature = "serde-support")]
+use serde::ser;
+#[cfg(feature = "serde-support")]
+use serde::ser::{Serialize, Serializer, SerializeSeq, SerializeTuple};
+#[cfg(feature = "serde-support")]
+use serde::de;
+#[cfg(feature = "serde-support")]
+use serde::de::{Deserialize, Deserializer, SeqAccess, Visitor};
+#[cfg(feature = "serde-support")]
+use glib::translate::from_glib_none;
+#[cfg(feature = "serde-support")]
+use gobject_ffi;
+
```

Comment:
It might be nicer to move this into a submodule inside this file, then you only need to use the `#cfg` once for the module

Reply:
I'll do that. Thanks.